crypto/evp_extra/evp_extra_test.cc

```diff
@@ -2427,6 +2471,18 @@ TEST_P(PerKEMTest, RawKeyOperations) {
   EXPECT_EQ(ERR_LIB_EVP, ERR_GET_LIB(err));
   EXPECT_EQ(EVP_R_OPERATION_NOT_SUPPORTED_FOR_THIS_KEYTYPE, ERR_GET_REASON(err));
 
+  void *tmp = (void*) pkey.get()->ameth;
+  pkey.get()->ameth = nullptr;
+  ASSERT_FALSE(EVP_PKEY_get_raw_public_key(pkey.get(), pk.data(), &pk_len));
+  err = ERR_get_error();
+  EXPECT_EQ(ERR_LIB_EVP, ERR_GET_LIB(err));
+  EXPECT_EQ(EVP_R_OPERATION_NOT_SUPPORTED_FOR_THIS_KEYTYPE, ERR_GET_REASON(err));
```

Comment:
NP: Is this the right error, does ERR_R_PASSED_NULL_PARAMETER make more sense? Or is this a carryover of existing behavior?